.travis.yml

```diff
@@ -1,15 +1,17 @@
 ---
+after_script:
```

Comment:
Alpha-sort makes sense, but in this case, logically speaking, so would order-by-chronological-execution (eg `before_script`->`script`->`after_script`).

Something to think about perhaps, but I think I'm mostly fine with either order in this type of case.